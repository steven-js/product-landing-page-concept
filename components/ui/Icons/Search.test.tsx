import { render } from '@testing-library/react'
import Search from './Search'

describe('Search icon test', () => {
  it('should render the Search icon unchanged', () => {
    const { container } = render(<Search />)
    expect(container).toMatchSnapshot()
  })
})
