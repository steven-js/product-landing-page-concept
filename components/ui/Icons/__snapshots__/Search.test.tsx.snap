// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search icon test should render the Search icon unchanged 1`] = `
<div>
  <svg
    fill="none"
    height="21"
    viewBox="0 0 21 21"
    width="21"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="m20.75 19.69-5.664-5.665a8.263 8.263 0 1 0-1.06 1.061l5.663 5.664 1.061-1.06ZM2 8.75a6.75 6.75 0 1 1 6.75 6.75A6.758 6.758 0 0 1 2 8.75Z"
      fill="var(--neutral-neutral-1000)"
    />
  </svg>
</div>
`;
